src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolutionResult.cs

```diff
@@ -1204,7 +1204,7 @@ private static bool HadLambdaConversionError(BindingDiagnosticBag diagnostics, A
                         symbols,
                         arg + 1,
                         argument.Display, //'<null>' doesn't need refkind
-                        UnwrapIfParamsArray(parameter, isLastParameter));
+                        new FormattedSymbol(UnwrapIfParamsArray(parameter, isLastParameter), SymbolDisplayFormat.CSharpErrorMessageTypeFormat));
```

Comment:
>CSharpErrorMessageTypeFormat [](http://example.com/codeflow?start=113&length=28)

I think I am starting to understand what we are trying to accomplish here, but the name of the option feels very confusing. It sounds to me as though it is meant to be used only with types.

Reply:
Can we change ```UnwrapIfParamsArray``` to always return type symbol and avoid the need to use special display format? We need type, let's use type.

Reply:
I guess we want to preserve ref-ness information.

Reply:
Yeah, I tried to change `UnwrapIfParamsArray` to return type at first. But it doesn't preserve ref-ness, as you say.